tasks/plugins.yml

```diff
@@ -25,7 +25,7 @@
     shell: umask 0022; sensu-install -p {{ item }}
     with_items: "{{ sensu_remote_plugins }}"
     changed_when: false
-    when: sensu_remote_plugins > 0
+    when: sensu_remote_plugins | length > 0
```

Comment:
Womp, so the integration tests fail with this:
```
fatal: [localhost]: FAILED! => {"msg": "The conditional check 'sensu_remote_plugins | length > 0' failed. The error was: Unexpected templating type error occurred on ({% if sensu_remote_plugins | length > 0 %} True {% else %} False {% endif %}): object of type 'NoneType' has no len()
```

Reply:
I believe https://github.com/sensu/sensu-ansible/blob/master/defaults/main.yml#L79 is a trick to make an undefined YAML key. 

Perhaps a more sane approach to this would be:
```
when:
  - sensu_remote_plugins is defined
  - sensu_remote_plugins | length > 0
```

Just an idea

Reply:
Here's the error I'm receiving:
```
fatal: [REDACTED]: FAILED! => {
    "failed": true,
    "msg": "The conditional check 'sensu_remote_plugins > 0' failed. The error was: Unexpected templating type error occurred on ({% if sensu_remote_plugins > 0 %} True {% else %} False {% endif %}): '>' not supported between instances of 'list' and 'int'\n\nThe error appears to have been in '/home/dan/git/profiler/playbooks/galaxy_roles/cmacrae.sensu/tasks/plugins.yml': line 24, column 5, but may\nbe elsewhere in the file depending on the exact syntax problem.\n\nThe offending line appears to be:\n\n\n  - name: Ensure any remote plugins defined are present\n    ^ here\n"
}
```
Output of "ansible --version"
```
ansible 2.4.0.0
  config file = /home/dan/git/profiler/playbooks/ansible.cfg
  configured module search path = ['/home/dan/.ansible/plugins/modules', '/usr/share/ansible/plugins/modules']
  ansible python module location = /home/dan/.pyenv/versions/3.6.2/envs/profiler@3.6.2/lib/python3.6/site-packages/ansible
  executable location = /home/dan/.pyenv/versions/profiler@3.6.2/bin/ansible
  python version = 3.6.2 (default, Dec 12 2017, 18:18:13) [GCC 7.2.1 20170915 (Red Hat 7.2.1-2)]
```
Maybe this is a problem only when running 3.6, I'll try with a 2.X version and see if I get the same result.